app_legacy/Helpers/database/forum.php

```diff
@@ -278,6 +300,10 @@ function submitTopicComment(
     sanitize_sql_inputs($user);
     $userID = getUserIDFromUser($user);
 
+    if (getIsForumDoublePost($userID, $topicID, $commentPayload)) {
+        return false;
```

Comment:
I'm not sure what this would look like for an actual double post, but manually entering the exact test from the previous post results in an error banner. Maybe it should silently fail?

Reply:
I agree it would be better to silently fail. Resolved in https://github.com/RetroAchievements/RAWeb/pull/1559/commits/e6b2669d350f032bf00e601c4fc5d98fe8ca0634.